dojo/settings/settings.dist.py

```diff
@@ -1457,6 +1458,7 @@ def saml2_attrib_map_format(dict):
     'KubeHunter Scan': DEDUPE_ALGO_HASH_CODE,
     'kube-bench Scan': DEDUPE_ALGO_HASH_CODE,
     'Threagile risks report': DEDUPE_ALGO_UNIQUE_ID_FROM_TOOL_OR_HASH_CODE,
+    'MSDefender API': DEDUPE_ALGO_HASH_CODE,
```

Comment:
`IndexError` where?
If some findings should be updated, because the new version of the report contains more updated data, this should resolved in reimporter.
If it not happening right now, it should be reviewed.
If we close old findings and reopen new ones, we are losing track of them. We do not know when was the finding discovered for the first time and the SLAs are useless.